spring_package/Modeller.py

```diff
@@ -39,7 +39,7 @@ def createMonomer(resultFile, identifier, pdbDatabase, outputName):
 def TMalign(fileA, fileB, tmName="temp/tmalign"):
     try:
         tmResult = open("%s.out" % tmName, "w")
-        subprocess.run(["tmalign", fileA, fileB, "-m", "%s.mat" % tmName], check=True, stdout=tmResult)
+        subprocess.run(["TMalign", fileA, fileB, "-m", "%s.mat" % tmName], check=True, stdout=tmResult)
```

Comment:
@guerler that is really important ... not sure how this can work on usegalaxy.org without that. The Conda package has a binary that is called `TMalign` and not `tmalign`...

Reply:
I tried both and it did not seem to be case sensitive. Thank you.

Reply:
Was that on OSX? This is the only thing that worked for me.

Reply:
Yes it was, thanks for catching this.